kubemarine/resources/schemas/definitions/services/kubeadm.json

```diff
@@ -5,7 +5,7 @@
   "properties": {
     "kubernetesVersion": {
       "type":  "string",
-      "default": "v1.26.11",
+      "default": "v1.27.13",
```

Comment:
according to `defaults.yaml`, we use `v1.28.12` as a default, so maybe we should use it in json schema too?

Reply:
Discussed with Sergey. It probably makes in scope of this activity to also change default version from 1.28.12 to 1.30.3 (current recommended).